src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerConfigTests.cs

```diff
@@ -2101,6 +2100,37 @@ public void GlobalConfigOptionsAreEmptyWhenNoGlobalConfig()
             Assert.Empty(globalOptions.TreeOptions);
         }
 
+        [Theory]
```

Comment:
>Theory [](start = 9, length = 6)

Consider testing:
"/path{"
"/path}"
"/path?"
"/path,"
"/path\\"
"/path\\\\"

Reply:
Consider testing:
"//path"
"//"
"/\\{\\}\\,\\[\\]\\*"

---
In reply to: [490319622](https://github.com/dotnet/roslyn/pull/47728#discussion_r490319622) [](ancestors = 490319622)